ci/install-manuscript-editor.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+## install-manuscript-editor.sh: TODO
+
+# Set options for extra caution & debugging
+set -o errexit \
+    -o pipefail
+
+pip install -U manubot-ai-editor
```

Comment:
If run-manuscript-editor is moved to the python package, then `manubot-ai-editor` can be as an extra dependency and could be installed like:

```
pip install manubot[openai]
```

Reply:
Perfect!